src/main/scala/uk/gov/nationalarchives/referencegenerator/Base25Encoder.scala

```diff
@@ -0,0 +1,17 @@
+package uk.gov.nationalarchives.referencegenerator
+
+import uk.gov.nationalarchives.oci.Alphabet.Alphabet
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+object Base25Encoder {
```

Comment:
Could you rename this to `object Encoder` to make it more generic, as the base may change at some point.